keylime/cert_utils.py

```diff
@@ -88,3 +94,40 @@ def verify_ek(ekcert, tpm_cert_store=config.get("tenant", "tpm_cert_store")):
 
     logger.error("No Root CA matched EK Certificate")
     return False
+
+
+def verify_ek_script(script, env, cwd):
+    if script is None:
+        logger.warning("External check script (%s) not specified", script)
```

Comment:
Not sure what the current direction is for keylime but consider the f-strings with {script} ?

Reply:
> consider the f-strings

That makes me wonder if we want this for logs or not. The logic behind `log.level("str", params)` (note that `%` is not used) is that interpolation only happens iff the call is made.  With f-strings we move out from lazy interpolation.

Not sure that is relevant, tho.